unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/ProjectSceneCardView.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using DCL.Builder;
+using DCL.Helpers;
+using TMPro;
+using UnityEngine;
+
+namespace DCL.Builder
+{
+
+
+    internal interface IProjectSceneCardView : IDisposable
+    {
+        /// <summary>
+        ///  Setting button pressed
+        /// </summary>
+        event Action<ProjectData> OnSettingsPressed;
+
+        /// <summary>
+        /// Data of the project card
+        /// </summary>
+        Scene scene { get; }
+
+        /// <summary>
+        /// Position of the context menu button
+        /// </summary>
+        Vector3 contextMenuButtonPosition { get; }
+
+        /// <summary>
+        /// This setup the scene data
+        /// </summary>
+        /// <param name="scene"></param>
+        /// <param name="outdated">If the scene is sync with the project</param>
+        void Setup(Scene scene, bool outdated);
+        
+        /// <summary>
+        /// Active the card
+        /// </summary>
+        /// <param name="active"></param>
+        void SetActive(bool active);
+    }
+
+    public class ProjectSceneCardView : MonoBehaviour, IProjectSceneCardView
+    {
+        const int THMBL_MARKETPLACE_WIDTH = 196;
+        const int THMBL_MARKETPLACE_HEIGHT = 143;
+        const int THMBL_MARKETPLACE_SIZEFACTOR = 50;
+        
+        public event Action<ProjectData> OnSettingsPressed;
+
+        public Scene scene => sceneData;
+        public Vector3 contextMenuButtonPosition { get; }
+        
+        [Header("Design Variables")]
+        [SerializeField] private  float animationSpeed = 6f;
+        
+        [Header("Project References")]
+        [SerializeField] private Texture2D defaultThumbnail;
+        
+        [Header("Prefab references")]
+        [SerializeField] private GameObject loadingImgGameObject;
+        [SerializeField] private GameObject outdatedGameObject;
+        
+        [SerializeField] internal TextMeshProUGUI sceneNameTxt;
+        [SerializeField] internal TextMeshProUGUI sceneCoordsTxt;
+        
+        [SerializeField] internal CanvasGroup canvasGroup;
+        
+        [Space]
+        [SerializeField] private RawImageFillParent thumbnail;
```

Comment:
Here (and in all places where you are using images that need to be loaded from any source) we can use the `ImageComponentView` that we have in our UI Components library and already has all the logic related to images loading (from sprite, from texture o from an url) and the parent fill adaptation. In this way you wouldn't have to manage the request of the url image in your code.

Reply:
I thought about the implementation of ImageComponentView for this one but since the lands and the scenes already are using this system I wanted to maintain it in the same way in order to maintain legibility and the same structure.

For future implementations I will definitively use the ImageComponentView